configdefinitions/src/vespa/onnx-evaluator.def

```diff
@@ -0,0 +1,28 @@
+# Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+namespace=onnx
```

Comment:
```suggestion
package=ai.vespa.modelintegration.evaluator.config
```

You also need to export the Java package from `configdefinitions` using `package-info.java`